debug-bar-elasticpress.php

```diff
@@ -89,3 +90,33 @@ function ep_setup_query_log() {
 	EP_Debug_Bar_Query_Log::factory();
 }
 add_action( 'plugins_loaded', 'ep_setup_query_log' );
+
+/**
+ * Display an admin notice if the minimum ElasticPress plugin version is not met.
+ */
+function maybe_display_admin_notice() {
```

Comment:
@burhandodhy this function name is too generic and may conflict with some other plugin. Can we please add a namespace to this file (and get rid of all the method prefixes?)